script/events/outside.js

```diff
@@ -91,6 +91,62 @@ Events.Outside = [
 			}
 		}
 	},
+	{/* The Shady Builder */
+		title: _('The Shady Builder'),
+		isAvailable: function() {
+			return Engine.activeModule == Outside && $SM.get('game.buildings["hut"]', true) >= 5 && $SM.get('game.buildings["hut"]', true) < 20;
```

Comment:
Thanks, that's a good idea.  I just made the changes and now the shady builder appears in the room instead of the village.